pipelines/skylab/multiome/Multiome.changelog.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.1
+2023-12-12 (Date of Last Commit)
+* Downgraded Cell Bender from v3.0.1 to v3.0.0
```

Comment:
Should this say v0.3.1 to v0.3.0?

Reply:
Yes I think so. This is the version were using now. https://raw.githubusercontent.com/broadinstitute/CellBender/v0.3.0/wdl/cellbender_remove_background.wdl

Reply:
Oh yes! I see it now

Reply:
Thanks for pointing that out!